CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+
+- [Monorepo setup](#monorepo-setup)
+- [Installation](#installation)
+- [Develop the CLI packages](#Develop-the-CLI-packages)
+- [Release the packages](#release-the-packages)
+- [Available commands](#commands)
+- [CI](#ci)
+- [Add a new package to the monorepo](#add-a-new-package-to-the-monorepo)
+
+## Monorepo setup
+
+This repository is managed as a monorepo that is composed of many npm packages.
+
+For more information on monorepo:
+
+- [Babel GitHub](https://github.com/babel/babel/blob/master/doc/design/monorepo.md)
+- [Shopify GitHub](https://github.com/Shopify/quilt/blob/master/Decision%20records/00%20-%20Use%20a%20Lerna%20monorepo.md)
+- [Google](https://www.google.com/search?q=monorepo)
+
+### PNPM workspace
+
+This monorepo is using PNPM workspace feature to handle the installation of the npm dependencies and manage the packages interdependencies.
+
+It's important to note that PNPM workspace will **hoist** the npm dependencies at the root of the workspace. This means that there might not be a *node_modules* directory nested in the packages directories. The npm dependencies are installed in a *node_modules* directory at the root of the workspace and a single *pnpm-lock.yaml* file is generated at the root of the workspace.
+
+## Installation
+
+This project uses PNPM workspace. Therefore, you must [install PNPM](https://pnpm.io/installation):
+
+To install the project, open a terminal at the root of the workspace and execute the following command:
+
+```bash
+pnpm i
+```
+
+## Develop the CLI packages
+
+The following documentation is a brief overview of the tools and processes involved in the development of the CLI packages.
+
+> To develop for each package, you can run `pnpm dev` from the project root folder. This will build all the content of the `packages` folder.
+
+### Working in `@workleap/create-project`
+
+This package can be found under the folder `packages/create-project`.
+
+To build it for development, from the folder, run `pnpm dev`. This will build the project in development mode and link the binary locally. Once done, you will be able to run it from a terminal by using the `create-project` command.
+
+When the package is built from the `pnpm dev` command, you can attach a debugger and debug directly from the TypeScript files.
+
+### Working in `@workleap/foundry`
+
+This package can be found under the folder `packages/foundry`.
+
+To build it for development, from the folder, run `pnpm dev`. This will build the project in development mode and link the binary locally. Once done, you will be able to run it from a terminal by using the `foundry` command.
+
+When the package is built from the `pnpm dev` command, you can attach a debugger and debug directly from the TypeScript files.
+
+### Linting
+
+To run lint on the packages, call `pnpm lint` from the project root folder.
+
+### Testing
+
+Once each package is built in dev mode (`pnpm dev`), you can run the unit test using `pnpm test`.
```

Comment:
It's kind of weird to me to run `pnpm dev` before executing the tests because usually a `dev` script is started in watch mode and therefore is a long running process.

Usually, test runners are capable of building the project on their own and then execute the tests.

Reply:
True, I mixed up the concepts of automatic tests (`pnpm test`) and manual testing (where `pnpm dev` is used) in this sentence... I will rewrite that part.